policyengine_us/parameters/gov/usda/snap/income/deductions/utility/limited/allowance.yaml

```diff
@@ -1,184 +1,235 @@
 IN:
   2010-01-01: 0
   2021-10-01: 259
+  2023-05-01: 276.0
 KS:
   2010-01-01: 0
   2021-10-01: 286
+  2023-10-01: 327.0
 KY:
   2010-01-01: 0
   2021-10-01: 281
+  2023-10-01: 395.0
 LA:
   2010-01-01: 0
   2021-10-01: 203
+  2023-10-01: 230.0
 MA:
   2010-01-01: 0
   2021-10-01: 421
+  2023-10-01: 520.0
 MD:
   2010-01-01: 0
   2021-10-01: 238
+  2023-01-01: 337.0
 ME:
   2010-01-01: 0
   2021-10-01: 285
+  2023-10-01: 341.0
 MI:
   2010-01-01: 0
   2021-10-01: 0
+  2023-10-01: 0.0
 MN:
   2010-01-01: 0
   2021-10-01: 0
+  2023-10-01: 0.0
 MO:
   2010-01-01: 0
   2021-10-01: 327
+  2023-10-01: 351.0
 MS:
   2010-01-01: 0
   2021-10-01: 206
+  2023-10-01: 191.0
 MT:
   2010-01-01: 0
   2021-10-01: 214
+  2023-10-01: 244.0
 NC:
   2010-01-01: 0
   2021-10-01: 331
+  2023-10-01: 532.0
 ND:
   2010-01-01: 0
   2021-10-01: 242
+  2023-10-01: 270.0
 NE:
   2010-01-01: 0
   2021-10-01: 267
+  2023-10-01: 303.0
 NH:
   2010-01-01: 0
   2021-10-01: 277
+  2023-10-01: 350.0
 NJ:
   2010-01-01: 0
   2021-10-01: 359
+  2023-10-01: 464.0
 NM:
   2010-01-01: 0
   2021-10-01: 135
+  2023-10-01: 215.0
 NV:
   2010-01-01: 0
   2021-10-01: 249
+  2023-10-01: 350.0
 NY:
   2010-01-01: 0
   2021-10-01: 285
+  2023-10-01: 332.0
 OH:
   2010-01-01: 0
   2021-10-01: 377
+  2023-10-01: 453.0
 OK:
   2010-01-01: 0
   2021-10-01: 292
+  2023-10-01: 334.0
 OR:
   2010-01-01: 0
   2021-10-01: 353
+  2023-10-01: 370.0
 PA:
   2010-01-01: 0
   2021-10-01: 317
+  2023-10-01: 401.0
 RI:
   2010-01-01: 0
   2021-10-01: 0
+  2023-10-01: 0.0
 SC:
   2010-01-01: 0
   2021-10-01: 216
+  2023-10-01: 247.0
 SD:
   2010-01-01: 0
   2021-10-01: 220
+  2023-10-01: 250.0
 TN:
   2010-01-01: 0
   2021-10-01: 136
+  2023-10-01: 164.0
 TX:
   2010-01-01: 0
   2021-10-01: 345
+  2023-10-01: 391.0
 UT:
   2010-01-01: 0
   2021-10-01: 274
+  2023-10-01: 382.0
 VA:
   2010-01-01: 0
   2021-10-01: 0
+  2023-10-01: 0.0
 VI:
   2010-01-01: 0
   2021-10-01: 0
+  2023-10-01: 0.0
 VT:
   2010-01-01: 0
   2021-10-01: 250
+  2023-10-01: 291.0
 WA:
   2010-01-01: 0
   2021-10-01: 361
+  2023-10-01: 383.0
 WI:
   2010-01-01: 0
   2021-10-01: 317
+  2023-10-01: 347.0
 WV:
   2010-01-01: 0
   2021-10-01: 280
+  2023-10-01: 305.0
 WY:
   2010-01-01: 0
   2021-10-01: 291
+  2023-10-01: 317.0
 metadata:
   unit: currency-USD
   period: month
   reference:
-    - name: USDA | Food and Nutrition Service | SNAP | Standard Utility Allowances
-      href: https://www.fns.usda.gov/snap/eligibility/deduction/standard-utility-allowances
-    - name: SNAP Screener | SNAP Eligibility Parameters
-      href: https://www.snapscreener.com/?p=table
+  - name: USDA | Food and Nutrition Service | SNAP | Standard Utility Allowances
+    href: https://www.fns.usda.gov/snap/eligibility/deduction/standard-utility-allowances
+  - name: SNAP Screener | SNAP Eligibility Parameters
+    href: https://www.snapscreener.com/?p=table
+  - name: USDA utility allowances by state spreadsheet. Extracted by build_utility_allowances()
```

Comment:
What's `build_utility_allowances`? Is it code we could include here? @nikhilwoodruff do we have a standard for including such code in the repo?

Reply:
build_utility_allowances() is the LLM-based scraper code I am working on.